src/main/java/ai/elimu/web/content/word/WordCreateController.java

```diff
@@ -125,6 +125,9 @@ public String handleSubmit(
         if (existingWord != null) {
             result.rejectValue("text", "NonUnique");
         }
+        if(word.getText().contains(" ")) {
```

Comment:
I suggest you use the `StringUtils` library instead, which is a good library for covering various edge cases.

Reply:
Sure, will look into that.